test/verify/check-services

```diff
@@ -778,7 +778,7 @@ ExecStart=/bin/true
         b.click(svc_sel("test.service"))
         b.wait_js_cond('window.location.hash === "#/test.service"')
 
-        b.wait_attr("#Requires a:contains('not-found.service')", "class", "disabled")
+        b.wait_attr("#Requires a:contains('not-found.service')", "disabled", "")
```

Comment:
This will most likely require a condition for rhel-8-2-distropkg. I triggered all the tests to see that.

Reply:
It most likely will need rebase first due to https://github.com/cockpit-project/cockpit/commit/4ace78c14b0600714e091a6a07e858b88807e170

Reply:
I've added the condition to keep the old behavior for distropkg.